src/Particle/LongRange/StructFact.cpp

```diff
@@ -23,8 +23,12 @@
 namespace qmcplusplus
 {
 //Constructor - pass arguments to KLists' constructor
-StructFact::StructFact(ParticleSet& P, RealType kc)
-    : DoUpdate(false), SuperCellEnum(SUPERCELL_BULK), StorePerParticle(false)
+StructFact::StructFact(const ParticleSet& P, RealType kc)
```

Comment:
Actual constructor arguments should be `SpeciesSet&, kc, ParticleLayout&`

Reply:
Actually, only needs the number of particles, the number of species and the lattice. Changed.